pandas/core/arrays/arrow/base.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import annotations
+
+import numpy as np
+import pyarrow as pa
+
+from pandas._typing import DtypeObj
+from pandas.util._decorators import cache_readonly
+
+from pandas.core.dtypes.base import StorageExtensionDtype
+
+from pandas.core.arrays.arrow import ArrowExtensionArray
+
+
+class ArrowDtype(StorageExtensionDtype):
```

Comment:
why are the dtypes here? shouldn't these be in pandas/core/dtypes ?

Reply:
I was thinking of consolidating all arrow related functionality under `/arrays/arrow/`. Thoughts @jbrockmendel

Reply:
no objection.  Might be worth keeping _something_ in core.dtypes.dtypes for dependency-structure purposes

Reply:
ok fair, looking to keep these separate as much as possible, maybe rename this to _dtype.py at some point.